README.md

```diff
@@ -16,6 +16,8 @@ It is designed to provide a delightful developer experience, enabling community
 
 ## Running rbuilder
 
+**Warning**: Even if they are rare, before running a builder you should be aware of [`reorg losses`](./REORG_LOSSES.md).
```

Comment:
I would not add this as the top info on "running rbuilder". 

It seems better to place this line at the _end_ of the "Live" subsection.